static/scripts/theme-switch.js

```diff
@@ -54,9 +70,8 @@ if (storageAvailable("localStorage")) {
 }
 if (savedTheme) {
     document.documentElement.setAttribute("data-theme", savedTheme);
-} else if (window.matchMedia("(prefers-color-scheme: dark)").matches) {
-    document.documentElement.setAttribute("data-theme", "dark");
```

Comment:
@apiraino Do you remember the purpose of this line? I removed it because once the theme preference is stored explicitly, the system preference is not followed anymore if the user changes it later on.

In my testing, I think there was a little bit more flickering without this. Meaning the page was initially white and then quickly flashed to dark according to my system preference. But there was some amount of flickering in both cases, hard to tell the difference.

I'm not sure why that would be the case, unless `window.matchMedia` takes a noticeable amount of time to run (Firefox) ?